### DISCUSSION TITLE
Removed the branch search box make many difficult to create the PR

### DISCUSSION THREAD

**Discussion by khoa-nv** [community/community#129280] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello Github Team,
Today, when I create a new PR I see that the search box to search the branch was removed. I think this brings many difficulties to creating a new PR. For example, my project has many branches and I want to create the PR to the develop branch instead of the master or main, I need to scroll and search by my selft.
![image](https://github.com/community/community/assets/54359582/5e5aa798-1810-4804-af74-244585aaf5ff)

**Comment by lemuelmc** [comment]
Please add the search function back.
